native/chat/chat-thread-background.react.js

```diff
@@ -34,10 +36,34 @@ export default function BackgroundChatThreadList(
     props.navigation.setOptions({ title });
   }, [props.navigation, unreadBackgroundThreadsNumber]);
 
+  const emptyItem = <EmptyItem />;
```

Comment:
The issue I raised from the last review is still happening:

> we're generating a new `React.Node` in `ChatThreadBackground` every time it renders, that means we will end up generating a new data for the `FlatList` in `ChatThreadList` every time `ChatThreadBackground` renders

Here's the solution I proposed in the last review:

> instead of passing in a `React.Node`, let's pass in a React component (`React.ComponentType`)

If this doesn't make sense, try Googling those two and seeing what the difference is. If it's still confusing, feel free to message me or @zrebcu411 and we can help explain.